google/cloud/forseti/scanner/scanners/config_validator_scanner.py

```diff
@@ -85,7 +83,7 @@ def _flatten_violations(self, violations):
                 'full_name': full_name,
                 'rule_index': 0,
                 'rule_name': violation.constraint,
-                'violation_type': self.VIOLATION_TYPE,
+                'violation_type': 'CV ' + violation.constraint,
```

Comment:
Slight nit- would suggest putting 'CV: ' + violation.constraint instead to make it more clear that it is from CV.

Reply:
Thanks for the suggestion! Violation type/category of finding in CSCC usually do not have a colon(:) in it. They have underscore(_) instead.